backend/src/articles_service/articles_parser.py

```diff
@@ -56,9 +58,14 @@ def get_keywords(self) -> list[str]:
                 keywords[i] = keywords[i].split(", ")
                 for a in range(len(keywords[i])):
                     keywords[i][a] = keywords[i][a].strip()
-            return keywords
+            return keywords[len(keywords)-1]
```

Comment:
why are we taking only last element? if there is a reason u can use [-1] and please change typing